### DISCUSSION TITLE
Generate Tag from path-substring

### DISCUSSION THREAD

**Discussion by Gewerd-Strauss** [SilentVoid13/Templater#942] [Help]
Hello, 

Assume I have a lecture note in `root/%path%/BE13 Physical Chemistry`, and want to insert a template. Said template contains a `tags:` yaml-field, and for lectures I tag the respective course header (in this case, `BE13`). Is there a way to extract said substring from the note's path? If not, is it doable from the notes' title? (This would be the less preferred method as not all notes in a course-folder also contain the course header in their title.)

Thank you.  
Sincerely,  
~Gw

**Comment by Zachatoo** [comment]
You can use [tp.file.folder](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/file-module.html#tpfilefolderrelative-boolean--false) to get the path, and then use JS to get the part of the path you want. You'll probably need the [split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) string method.
